jib-maven-plugin/src/test/resources/projects/simple/pom-no-to-image.xml

```diff
@@ -0,0 +1,50 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.test</groupId>
+  <artifactId>my-artifact-id</artifactId>
+  <version>1</version>
+
+  <name>Name not Usable as Image Reference</name>
```

Comment:
To not have to have another pom, what about just setting the `simple/pom.xml`'s `<name>`?

Reply:
`simple/pom.xml` defines `<to><image>`, so I had to create a new one without it. I think this is simpler.

Reply:
Ah okay.